test/e2e/playwright/fixtures/index.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+import { Identity } from "@ory/kratos-client"
+import { test as base, expect } from "@playwright/test"
+import { OryKratosConfiguration } from "../../cypress/support/config"
+import { merge } from "lodash"
+import { default_config } from "../setup/default_config"
+import { writeFile } from "fs/promises"
+import { faker } from "@faker-js/faker"
+
+// from https://stackoverflow.com/questions/61132262/typescript-deep-partial
+type DeepPartial<T> = T extends object
+  ? {
+      [P in keyof T]?: DeepPartial<T[P]>
+    }
+  : T
+
+type TestFixtures = {
+  identity: Identity
+  configOverride: DeepPartial<OryKratosConfiguration>
+  config: void
+}
+
+type WorkerFixtures = {}
+
+export const test = base.extend<TestFixtures, WorkerFixtures>({
+  configOverride: {},
+  config: [
+    async ({ request, configOverride }, use) => {
+      const configToWrite = merge(default_config, configOverride)
+
+      const resp = await request.get("http://localhost:4434/health/config")
+
+      const configRevision = await resp.body()
+
+      await writeFile(
+        "playwright/kratos.config.json",
```

Comment:
This isn't thread safe - meaning that we can't execute playwright tests in parallel. Is parallelization disabled in playwright?

Reply:
Yes - no parallelization, there is no way to make this thread safe with the current config model of Kratos.